launcher/ui/pages/global/MinecraftPage.h

```diff
@@ -39,6 +39,7 @@
 #include <memory>
 
 #include <Application.h>
```

Comment:
Me evil I missed this one:
```suggestion
#include "Application.h"
```